cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -156,7 +175,7 @@ func (p PasstNetworkConfigurator) generateInterface() (*domainschema.Interface,
 	}
 
 	const (
-		ifaceTypeUser     = "user"
+		ifaceTypeUser     = "vhostuser"
```

Comment:
nit: The const used to represent the `user` type, now that the type is `vhostuser` its name is no longer accurate.
Please consider `ifaceType` or something along those lines.